src/content/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data.mdx

```diff
@@ -50,244 +50,226 @@ You can use tags in the UI to filter down to relevant data. Here is an example:
+- Identify teams, roles, environments, or regions to know who's responsible for what. See [some examples](#org-examples).
+- Better organize and search dashboards and [workloads](/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster).
+- [Query and chart APM data](#query-apm-tags).
 
 Tags are useful for organizing data at a high level. If you want to add more fine-grained detail, like capturing user names or other high-cardinality values, [custom attributes](/docs/using-new-relic/data/customize-data/collect-custom-attributes) or [custom events](/docs/insights/insights-data-sources/custom-data/report-custom-event-data) are a better solution.
 
-<Callout variant="tip">
-  Tags were previously called `labels` in the New Relic UI. You may sometimes see `label`in our code; for example, some of our APM agent config files use a `label` config option for tags.
-</Callout>
+## Managing tags [#tag-management]
 
-## Tag format and limits [#formatting]
+Some important [attributes](/docs/using-new-relic/welcome-new-relic/get-started/glossary#attribute) are available by default to be used as tags. For example, the account name, the account ID, and some basic host and server metadata. You can't remove these available-by-default tags.
 
-Format requirements and limits for tags:
+When you add tags via the UI or API, this occurs at the entity level. This means that, for example, if you have multiple APM agents monitoring a single entity and apply a tag via the UI, that tag is available for all data coming from that entity.
 
-* The UI has limits on the total number of tags it can display per entity:
-  * Limit of 100 for tags reported by default or added via agent/integration configuration.
-  * Limit of 100 for custom tags added via UI or API.
-* The following are maximum character length for tags:
-  * Key: 128 characters
-  * Value: 256 characters
-* Allowed characters:
-  * Characters must be UTF-8.
-  * When [using NerdGraph to add tags](/docs/apis/nerdgraph/examples/nerdgraph-tagging-api-tutorial), a dash (`-`) in a tag key is interpreted as a minus symbol. If your tag key has a dash, use back ticks around it (like `` `key-name` ``).
+### Add tags [#add-tags]
 
-## Best practices and tips [#tag-best-practices]
+1. Go to [one.newrelic.com](https://one.newrelic.com) and select a monitored entity, like a monitored app or host.
+2. Near the entity's name at the top of the page, select the **See metadata and manage tags** menu.
+3. In the **Add a new tag...** field of the pop-up menu, add a tag with this format: `accountId: 123456789`.
+  <img
+  title="Add tags in New Relic"
+  alt="Add tags in New Relic"
+  src={tagManagement}
+  />
 
-Before [adding tags](#tag-sources), we recommend first seeing how our default tags work for you. When you're ready to add your own tags, review the following best practices tips to get the most out of them.
+4. For APM agents, restart your service.
```

Comment:
I think the 5th point will be a bit confusing for the users since you don't need to restart your service in order to see the tags in the UI (if you added them via UI or GraphQL). The restart is needed only if you want to use those tags in NRQL queries.

Reply:
I'll add that remark then!